### PR TITLE
Jobs debug api readiness

### PR DESCRIPTION
**Pull request by tisnik**
I'd like to create some tests for jobs debug API, so first we would need to be sure the API is running AND the DB is prepared as well (please see https://github.com/fabric8-analytics/fabric8-analytics-jobs/issues/87 why this step is needed here).

### CODE REVIEW

File: integration-tests/features/environment.py [hunk 1]
```diff
@@ -200,6 +203,18 @@ def _wait_for_system(context, wait_for_server=60):
                         format(s=wait_for_server))
 
 
+def _wait_for_jobs_debug_api_service(context, wait_for_service=60):
+    start = datetime.datetime.now()
+    wait_till = start + datetime.timedelta(seconds=wait_for_service)
+
+    while datetime.datetime.now() < wait_till:
```
Comment (fridex) [inline integration-tests/features/environment.py]: You can directly do:

```python
for _ in wait_for_service:
    if _is_jobs_debug_api_running(context):
        break
     time.sleep(1)
else:
    raise Exception('Timeout waiting for jobs debug API service')
```

So there is no need to use datetime and compute delta.

Comment (jpopelka) [inline integration-tests/features/environment.py]: > for _ in wait_for_service:

`wait_for_service` is int so not iterable

Comment (fridex) [inline integration-tests/features/environment.py]: > wait_for_service is int, so not iterable

+1, missing `range()`

Comment (tisnik) [inline integration-tests/features/environment.py]: @fridex do you mean: for _ in range(wait_for_service)? given the sleep is set to 1, it's nearly the same, thx

Comment (fridex) [inline integration-tests/features/environment.py]: yes, exactly

**Pull request merged**